server/WebAPI/Controllers/ForgotPasswordController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using WebAPI.DTO;
+using WebAPI.Services;
+
+namespace WebAPI.Controllers
+{
+  [Authorize]
+  [Route("api/[controller]")]
+  [ApiController]
+  public class ForgotPasswordController : ControllerBase
+  {
+    private readonly ResetPasswordService resetPasswordService;
+    public ForgotPasswordController(ResetPasswordService resetPasswordService)
+    {
+      this.resetPasswordService = resetPasswordService;
+    }
+
+    [HttpPost("ResetPassword")]
+    public IActionResult ResetPassword(ResetPasswordModel resetPasswordModel)
+    {
+      if(resetPasswordModel.Email != "" && resetPasswordModel.Password != "")
```

Comment:
do we need any checks there or just any user can reset password for any other user? it is security breach then

Reply:
I don't even know how to fix it

Reply:
lets continue this discussion here
public bool UpdateUserPassword(string email, string newPassword)

as its the same flow. in short, you'll need to check that email in this "resetPasswordModel" matches email of currently logged in user.